src/game/client/draw/w3ddefaultdraw.cpp

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief W3D Default Draw Module
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "w3ddefaultdraw.h"
+#include "assetmgr.h"
+#include "drawable.h"
+#include "w3ddisplay.h"
+#include "w3dscene.h"
+#include "w3dshadow.h"
+
+W3DDefaultDraw::W3DDefaultDraw(Thing *thing, ModuleData const *module_data) :
+    DrawModule(thing, module_data), m_renderObject(nullptr), m_shadow(nullptr)
+{
+    if (!Get_Drawable()->Get_Template()->Get_LTA_Name().Is_Empty()) {

```

Comment:
store drawable and template so its less of a -> chain, annoying to read...

Reply:
fixed.